### DISCUSSION TITLE
Possibility to make an estimate

### DISCUSSION THREAD

**Discussion by bdlanduyt** [AndresMorelos/InvoncifyReleases#83] [General]
Is it possible to make an estimate/quote instead of an invoice?

**Comment by AndresMorelos** [comment]
@bdlanduyt  I had implemented a quote template, you can find it at the preview in the template setting.

![CleanShot 2022-06-06 at 08 45 32](https://user-images.githubusercontent.com/6563162/172173079-8b1b6ef5-298d-4649-a739-7f4da430be80.png)
